sonar-scm-git-plugin/pom.xml

```diff
@@ -41,7 +41,7 @@
     <dependency>
       <groupId>org.eclipse.jgit</groupId>
       <artifactId>org.eclipse.jgit</artifactId>
-      <version>5.2.0.201812061821-r</version>
+      <version>5.3.0.201903130848-r</version>
```

Comment:
I usually create a ticket for that, pointing to release notes so that support could investigate issues. See for example: https://jira.sonarsource.com/browse/SONARSCGIT-37

Reply:
Good point